src/crimson/os/seastore/onode_manager/staged-fltree/tree_utils.h

```diff
@@ -147,8 +152,17 @@ class KVPool {
     std::random_device rd;
     for (unsigned i = range2.first; i < range2.second; ++i) {
       for (unsigned j = range1.first; j < range1.second; ++j) {
-        auto ns_size = (unsigned)str_sizes[rd() % str_sizes.size()];
-        auto oid_size = (unsigned)str_sizes[rd() % str_sizes.size()];
+        unsigned ns_size;
```

Comment:
unsigned?  could this be uint32_t, please?

Reply:
Sure.
This file is for test/perf-tool purposes, so I didn't use types very carefully.

Reply:
fixed.